proto/nodeinstance/create/v1/creation.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
+option go_package = "node_instance/creation/v1;node_instancecreation";
+
+package nodeinstance.create.v1;
+
+message CreateNodeInstance {
+    string claim_id = 1;
```

Comment:
can we add comments to ensure claim_id of agent sending or noted being created.

Reply:
Added couple comments Timo, LMK if this covers what you had in mind :)